params/config.go

```diff
@@ -277,6 +282,11 @@ func (c *ChainConfig) IsLondon(num *big.Int) bool {
 	return isForked(c.LondonCompatibleBlock, num)
 }
 
+// IsEthTxType returns whether num is either equal to the ethTxType block or greater.
+func (c *ChainConfig) IsEthTxType(num *big.Int) bool {
```

Comment:
I think this function name feels little unnatural.. ! 
The name of the function is not what it actually does.

Reply:
I agree with you. How about `IsEthTxTypeHFEnabled`?

Reply:
That sounds good :)

Reply:
Appiled. Thanks